web/src/main.ts

```diff
@@ -1,10 +1,12 @@
 import 'windi.css';
+import 'floating-vue/dist/style.css';
```

Comment:
You can disable the eslint - dist for this line. Maybe you can try to add this to the component as well.